.ci/Makefile

```
@@ -0,0 +1,164 @@
+		--extra-vars "runId=$(RUN_ID) workspace=$(PROJECT_DIR)/ sshPublicKey=$(SSH_KEY_PUBLIC)" \
+		--ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
+		-t setup-stack \
+		-i $(STACK_IP_ADDRESS),
+
+.PHONY: create-stack
+create: provision-stack setup-stack
+
+.PHONY: destroy-stack
+destroy-stack:
+	$(VENV_DIR)/bin/ansible-playbook $(PROJECT_DIR)/.ci/ansible/playbook.yml \
+		--private-key="$(SSH_KEY)" \
+		--extra-vars="nodeLabel=stack nodeImage=$(STACK_IMAGE) nodeUser=$(STACK_IMAGE)" \
+		--extra-vars="runId=$(RUN_ID) workspace=$(PROJECT_DIR)/ sshPublicKey=$(SSH_KEY_PUBLIC)" \
+		--ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
+		-t destroy
+	rm -fr $(PROJECT_DIR)/.ci/stack-host-ip
+
+.PHONY: ssh-stack
+ssh-stack:
+	ssh -i $(SSH_KEY) $(STACK_USER)@$(STACK_IP_ADDRESS) -vvv
+
+.PHONY: provision-node
+provision-node: .runID
+	@:$(call check_defined, RUN_ID, You need to an unique RUN_ID. To create it please run 'make .runID' goal)
+	@:$(call check_defined, STACK_IP_ADDRESS, IP address of the stack)
+	$(VENV_DIR)/bin/ansible-playbook $(PROJECT_DIR)/.ci/ansible/playbook.yml \
+		--private-key="$(SSH_KEY)" \
+		--extra-vars "stackRunner=$(STACK_IP_ADDRESS) nodeLabel=$(NODE_LABEL) nodeImage=$(NODE_IMAGE) nodeInstanceType=$(NODE_INSTANCE_TYPE) nodeUser=$(NODE_USER)" \
+		--extra-vars "runId=$(RUN_ID) workspace=$(PROJECT_DIR)/ suite=$(SUITE) sshPublicKey=$(SSH_KEY_PUBLIC)" \
+		--ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
+		-t provision-node
+
+.PHONY: setup-node
+setup-node: .runID
+	@:$(call check_defined, RUN_ID, You need to an unique RUN_ID. To create it please run 'make .runID' goal)
+	@:$(call check_defined, STACK_IP_ADDRESS, IP address of the stack)
+	@:$(call check_defined, NODE_IP_ADDRESS, IP address of the test node)
+	$(VENV_DIR)/bin/ansible-playbook $(PROJECT_DIR)/.ci/ansible/playbook.yml \
+		--private-key="$(SSH_KEY)" \
+		--extra-vars "stackRunner=$(STACK_IP_ADDRESS) nodeLabel=$(NODE_LABEL) nodeImage=$(NODE_IMAGE) nodeInstanceType=$(NODE_INSTANCE_TYPE) nodeUser=$(NODE_USER)" \
+		--extra-vars "runId=$(RUN_ID) workspace=$(PROJECT_DIR)/ suite=$(SUITE) sshPublicKey=$(SSH_KEY_PUBLIC)" \
+		--ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
+		-t setup-node \
+		-i $(NODE_IP_ADDRESS),
+
+.PHONY: create-node
+create: provision-node setup-node
+
+.PHONY: destroy-node
+destroy-node:
+	$(VENV_DIR)/bin/ansible-playbook $(PROJECT_DIR)/.ci/ansible/playbook.yml \
+		--private-key="$(SSH_KEY)" \
+		--extra-vars="nodeLabel=$(NODE_LABEL) nodeImage=$(NODE_IMAGE) nodeUser=$(NODE_USER)" \
+		--extra-vars="runId=$(RUN_ID) workspace=$(PROJECT_DIR)/ sshPublicKey=$(SSH_KEY_PUBLIC) \
+		--ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
+		-t destroy
+	rm -fr $(PROJECT_DIR)/.ci/node-host-ip
+
+.PHONY: ssh-node
+ssh-node:
+	ssh -i $(SSH_KEY) $(NODE_USER)@$(NODE_IP_ADDRESS) -vvv
```

Same as the other one
```suggestion
	ssh -i $(SSH_KEY) $(NODE_USER)@$(NODE_IP_ADDRESS)
```